buurbak/src/app/Components/Footer.tsx

```diff
@@ -0,0 +1,130 @@
+"use client";
+
+import { ArchiveBoxXMarkIcon } from "@heroicons/react/24/outline";
+import Image from "next/image";
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+import Logo from "../Assets/buurbakLogoBigColor.svg";
+import LogoM from "../Assets/buurbakLogoMediumColor.svg";
+
+const Footer = () => {
+  const currentRoute = usePathname();
+
+  const pages = [
+    { title: "Home", link: "/" },
+    { title: "Aanbod", link: "/aanbod" },
+    { title: "Verhuren", link: "/verhuren" },
+    { title: "Contact", link: "/contact" },
+    { title: "Veel gestelde vragen", link: "/faq" },
+    { title: "Inloggen", link: "/inloggen" },
+  ];
+  const subPages = [
+    { title: "Privacy policy", link: "/" },
+    { title: "Algemene voorwaarden", link: "/" },
+    { title: "©2024", link: "/" },
+  ];
+
+  return (
+    <div>
+      <div
```

Comment:
Eigenlijk moet je niet twee verschillende versies van de pagina maken voor mobile en desktop. Je moet in de Divs zelf met gebruik van "md:" de code zo schrijven dat die voor zowel mobile als desktop werkt. Nu ben je nog steeds dubbele code aan het schrijven

Reply:
Ja voor de design die er zijn gekozen kwam er daar niet goud uit dus leek dit mij de beste optie

Reply:
het is echt flink bad practice om zo de code te doen, wat is er precies wat niet gewoon met md: kan?

Reply:
We zullen het hier morgen wel ff over hebben online of fysiek ligt er aan waar jij bent